### PR TITLE
Convert checksum/sha256 from base64 to hex at dependency endpoint

### PR DESCRIPTION
**Pull request by sonalkr132**
Currently bundler-api uses hex version.

### CODE REVIEW

**Comment by dwradcliffe** [comment]
Seems that the bundler-api database currently has a mix of both, but we're standardizing on hex now.

cc: @indirect @segiddins

File: app/models/version.rb [hunk 1]
```diff
@@ -315,7 +315,11 @@ def authors_array
   end
 
   def sha256_hex
-    sha256.unpack("m0").first.unpack("H*").first if sha256
+    Version._sha256_hex(sha256) if sha256
+  end
+
+  def self._sha256_hex(raw)
```
Comment (arthurnn) [inline app/models/version.rb]: why there is a `_` in front of this method?

Comment (sonalkr132) [inline app/models/version.rb]: To differentiate it from [`Versions#sha256_hex`](https://github.com/rubygems/rubygems.org/pull/1305/files#diff-3c35bc805bbb122cee7af2e3884a4d66R317)

Comment (arthurnn) [inline app/models/version.rb]: however one method is a class method and the other is an instance one. So I guess they could have the same name.

Comment (sonalkr132) [inline app/models/version.rb]: It is more about aesthetic and readability than syntax issue. Open an issue about why is this unnecessary and I will send a PR :wink:

**Pull request merged**